sdk-python/omnilogger/openai_listener.py

```diff
@@ -3,19 +3,27 @@
 import openai
 from prisma.types import llm_logsCreateInput
 
+from .calculator import cost_calculator
 from .database import send_to_db
 
 
 class CustomCompletion(openai.Completion):
     @classmethod
     def create(cls, model: str, prompt: str, **kwargs):
         result = super().create(model=model, prompt=prompt, **kwargs)
-        log = llm_logsCreateInput(
-            input_string=prompt,
-            output_string=result["choices"][0]["text"],
-            datetime_utc=datetime.utcfromtimestamp(result["created"]),
-            total_tokens=result["usage"]["total_tokens"],
-        )
+        input_tokens = result["usage"]["prompt_tokens"]
+        output_tokens = result["usage"]["completion_tokens"]
+        cost = None
+        try:
+            cost = cost_calculator("openai", model, input_tokens, output_tokens)
+        finally:
+            log = llm_logsCreateInput(
```

Comment:
Are we using camelCase or this_type_of_case for functions? We're doing both here